src/codegeneration/InstantiateModuleTransformer.js

```diff
@@ -174,8 +174,11 @@ class InsertBindingAssignmentTransformer extends ScopeTransformer {
  */
 export class InstantiateModuleTransformer extends ModuleTransformer {
 
-  constructor(identifierGenerator) {
-    super(identifierGenerator);
+  constructor(identifierGenerator, reporter, options) {
+    super(identifierGenerator, reporter, options);
+
+    this.anonymousModule =
+        options && !options.bundle && options.moduleName !== true;
```

Comment:
I've just been copying the pattern in https://github.com/google/traceur-compiler/blob/master/src/codegeneration/AmdTransformer.js#L35.

So you just want to add `undefined` as a default? Isn't that the natural behaviour though?